### PR TITLE
Elasticsearch cookbook v2

### PR DESCRIPTION
**Pull request by eherot**
Updates to make compatible with v2 of Elasticsearch cookbook

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -12,9 +12,9 @@
 depends 'java', '~> 1.0'
 depends 'runit', '~> 1.5'
 
-depends 'elasticsearch', '~> 0.3.11'
+depends 'elasticsearch', '~> 2.1'
```
Comment (jeffbyrnes) [inline metadata.rb]: Alt text

File: recipes/elasticsearch.rb [hunk 2]
```diff
@@ -4,63 +4,48 @@
...
-# this should be moved to the logserver cookbook
-node.set['elasticsearch']['discovery']['search_query'] =
-  "recipes:et_elk\\:\\:server AND chef_environment:#{node.chef_environment}"
-
-node.set['elasticsearch']['custom_config']['index.number_of_shards'] = 3
-node.set['elasticsearch']['custom_config']['indices.memory.index_buffer_size'] = '50%'
-node.set['elasticsearch']['custom_config']['index.translog.flush_threshold_ops'] = 50_000
+#
+# -***** NEW RECIPE STARTS HERE ****-
+#
```
Comment (jeffbyrnes) [inline recipes/elasticsearch.rb]: This seems unnecessary

**Comment by jeffbyrnes** [comment]
:+1:

File: metadata.rb (hunk 1 above)
Comment (eherot) [inline metadata.rb]: Is understatement.

**Comment by EdHurtig** [comment]
cool

**Pull request merged**